docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+services:
+  mongodb:
+    image: mongo
+    container_name: mongodb
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=${MONGO_INITDB_ROOT_USERNAME}
+      - MONGO_INITDB_ROOT_PASSWORD=${MONGO_INITDB_ROOT_PASSWORD}
+    ports:
+      - 27017:27017
```

Comment:
If we only want to access the database from the node application, we could consider communicating via the contains so we don't have to expose the port. But we can also leave it for convenience since we can still block the port in the firewall.

Reply:
I would like to have it that way while developing for convenience the firewall idea sounds good for the production server